spec.md

```diff
@@ -582,6 +604,12 @@ message ControllerPublishVolumeRequest {
   // Whether to publish the volume in readonly mode. This field is
   // REQUIRED.
   bool readonly = 5;
+
+  // OPTIONAL field for storing principal information to be used in
+  // the publishing of this volume. If the principal is present the
+  // plugin MAY ensure the volume published will be accessible for
+  // the herein specified entities only.
+  Principal container_acts_as = 6;
```

Comment:
`container_acts_as` is a confusing name given the "perspective" from which the other fields were named (a volume's, not a container's). Please pick a name from the perspective of the volume not that of the container, for example, `accessible_by` or some such